CMPUT301Project/app/src/main/java/com/example/GroupProject/AddHabitEvent.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2021-2022. Group 43 CMPUT301 F2021
+ * All rights reserved.
+ */
+
+package com.example.GroupProject;
+
+import android.content.Intent;
+import android.media.Image;
+import android.os.Bundle;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.ToggleButton;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.google.firebase.Timestamp;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.SetOptions;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AddHabitEvent extends AppCompatActivity {
+
+    EditText title;
+    EditText comment;
+    ImageButton goBack;
+    ImageButton confirm;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.add_habit_event);
+
+        Intent intent = getIntent();
+        Habit habit = (Habit) intent.getSerializableExtra("HABIT");
+        title = findViewById(R.id.event_title);
+        comment = findViewById(R.id.event_comment);
+        goBack = findViewById(R.id.go_back1);
+        confirm = findViewById(R.id.confirm_1);
+
+        FirebaseFirestore db;
+        db = FirebaseFirestore.getInstance();
+
+        confirm.setOnClickListener(view -> {
+            Map<String, Object> habitEvent = new HashMap<>();
+
+            // Add all these to Firestore
+            String eventTitle = title.getText().toString();
+            String eventComment = comment.getText().toString();
+
+            habitEvent.put("title", eventTitle);
+            habitEvent.put("comment", eventComment);
+
+            db.collection("Users")
+                    .document("John Doe")
```

Comment:
Change this to get logged in user instead of default "John Doe". Check Kyle's merged PR on how to do this.